entrypoint.d/45-geoip

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+if [[ "$GEOIP_ACCOUNT_ID" != "" && "$GEOIP_LICENSE_KEY" != "" ]] ; then
+    set -e
+    # see https://dev.maxmind.com/geoip/geoipupdate/
+    log INFO "Activating GeoIP/GeoLite2 updates"
+    log INFO "This product includes GeoLite2 data created by MaxMind, available from https://www.maxmind.com"
+    sed 's/^AccountID\s.*$/AccountID '"$GEOIP_ACCOUNT_ID"'/g;s/^LicenseKey\s.*$/LicenseKey '"$GEOIP_LICENSE_KEY"'/g' /etc/GeoIP.conf > /tmp/GeoIP.conf
```

Comment:
Add -i and sed replaces in place AFAIK.

```suggestion
    sed -i 's/^AccountID\s.*$/AccountID '"$GEOIP_ACCOUNT_ID"'/g;s/^LicenseKey\s.*$/LicenseKey '"$GEOIP_LICENSE_KEY"'/g' /etc/GeoIP.conf
```

Then you can remove the next 2 lines also.

Reply:
well, i had `-i` here before, with `-i` sed tries to create a temp file in the directory the file is in. With /etc/ this does not work. I found no convenient way to convince sed to place the temporary file in a writeable directory. So this was the suggested workarround for the problem.

Reply:
Oh ok then, thanks for the explanation.